### DISCUSSION TITLE
Issue with Email Delivery through SMTP Server Service

### DISCUSSION THREAD

**Discussion by santosh0798** [emailjs-com/help#27] [Q&A]
I am writing to seek assistance regarding an issue I am experiencing with the SMTP Server Service from EmailJS.

Currently, I am able to successfully send emails using the regular Gmail account integration in EmailJS. However, when I attempt to utilize the SMTP Server Service, although the emails are marked as sent successfully, they are not being received in the respective recipient email accounts.

I have ensured that I configured the SMTP server settings accurately in my EmailJS account, including the correct SMTP server address, port, username, password, and other authentication details. I have also checked the spam or junk folders of the recipient email accounts to ensure the emails are not being filtered incorrectly.

To troubleshoot the issue further, I would appreciate any guidance or assistance you can provide. Specifically, I would like to:

    Confirm that the SMTP server settings in my EmailJS account are correct and appropriately configured.
    Request any insights or error logs available regarding the email delivery process through the SMTP Server Service.
    Seek recommendations on potential solutions or workarounds to ensure reliable email delivery using the SMTP Server Service.

I highly value the functionality provided by EmailJS and its SMTP Server Service, and I am eager to resolve this issue promptly. Thank you in advance for your attention and assistance in resolving this matter. Please let me know if any additional information or details are required from my end.

**Comment by xr0master** [comment]
The OK means that the email has been verified and successfully transferred to your service. And your service has accepted the processing request. Unfortunately, we have no control over the last steps.

Can you look at the dashboard of your SMTP server if there are any unsent emails? This can happen for a variety of reasons. For example, the recipient's address was not correct. It is also possible that the email ends up in a SPAM folder.​

Please make sure you check the mailbox of the address you have specified in your template as the recipient.

**Comment by lkbrian** [comment]
I'm not able to create an smtp service i get a 403 yet i recieve the test mail to my domain email.
a 403 that the service isn't created

**Reply by xr0master** [reply]
Please ensure cookies are allowed for the dashboard in your browser.
